src/js/components/awardv2/idv/activity/ActivityChartTooltip.jsx

```diff
@@ -80,8 +80,8 @@ export default class IdvActivityTooltip extends React.Component {
     }
 
     truncateText(awardingAgencyNameDiv, recipientNameDiv, theTooltipWidth) {
-        const { graphWidth, recipientName, awardingAgencyName } = this.props.data;
-        if (this.props.data.graphWidth < theTooltipWidth) {
+        return new Promise((resolve) => {
```

Comment:
Tested this locally and it appears we don't need to return a promise, we can just use the async / await on line 159. `setState` itself is async so I don't think we need to wrap all this in a promise. Maybe I'm missing something but that's my understanding.

Reply:
Not sure we really want to do that considering https://stackoverflow.com/questions/47019199/why-does-async-await-work-with-react-setstate